### DISCUSSION TITLE
Unsupport language handling? should go back to default language?

### DISCUSSION THREAD

**Discussion by KhantMin-coder** [amannn/next-intl#227] [Q&A]
### Description

Hi
I am following the Nextjs 13, client component [translations.](https://next-intl-docs.vercel.app/docs/next-13/client-components)

My url is something like this `http://localhost:3000/en?token=e8dcef6d`

Its works fine since i follow the example my app support `en` and `de`

<img width="453" alt="Screenshot 2023-03-27 at 2 55 21 PM" src="https://user-images.githubusercontent.com/56862118/227884831-32412ff3-7a0f-489c-b80e-09dc63fc8d0b.png">

but the problem is when the url goes into something like this, unsupported language. Eg (fr)

`http://localhost:3000/fr?token=e8dcef6d`

<img width="519" alt="Screenshot 2023-03-27 at 2 55 35 PM" src="https://user-images.githubusercontent.com/56862118/227884989-9c9142f1-5a76-4fa1-a026-530bdaeed472.png">

its completely break the translations.

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://github.com/amannn/next-intl/tree/main/packages/example-next-13

need to comment out `notFound()` in the head.tsx and layout.tsx

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: … 

### Expected behaviour

If the change the locale to `fr` which is the unsupported format. it should go back to english

example

`http://localhost:3000/fr?token=e8dcef6d`

**Comment by amannn** [comment]
> If the change the locale to fr which is the unsupported format. it should go back to english
> need to comment out notFound() in the head.tsx and layout.tsx

You can implement this by replacing `notFound` with `redirect`, right?

Please note that this is not a bug but a usage question, I'm converting this to a discussion.
